chia/util/memory_profiler.py

```diff
@@ -0,0 +1,245 @@
+from __future__ import annotations
+
+import asyncio
+import logging
+import pathlib
+import time
+import tracemalloc
+from typing import Dict, List, Optional, Set
+
+from chia.util.path import path_from_root
+
+
+async def mem_profile_task(root_path: pathlib.Path, service: str, log: logging.Logger) -> None:
+
+    profile_dir = path_from_root(root_path, f"memory-profile-{service}-{int(time.time())}")
```

Comment:
```suggestion
    profile_dir = path_from_root(root_path, f"memory-profile-{service}") / datetime.now().strftime("%Y-%m-%d_%H:%M:%S")
```

Maybe ^ ? This way its easier to understand when the profiling happened only by looking at the folder name and also it collects the results inside `memory-profile-<service>` and doesn't clutter the root dir.

Reply:
I'm actually increasingly preferring overwriting the previous profiles. it makes it simpler to reuse command lines in your shell history, and avoid entering new timestamps for every run.

Reply:
Hm, i think both ways have its pros, right. However, if we overwrite we should at least clear the old results on start. Im not sure whats the best but i feel like keeping old results might be a good thing.